elastic/logstash/pipeline/logstash.conf

```diff
@@ -15,126 +17,197 @@ filter {
+              "[mine_data][number]" => "%{[mine_lookup_result][0][number]}"
+              "[mine_data][name]" => "%{[mine_lookup_result][0][name]}"
+              "[mine_data][location][lat]" => "%{[mine_lookup_result][0][lat]}"
+              "[mine_data][location][lon]" => "%{[mine_lookup_result][0][lon]}"
+            }
+          }
+
+
+          mutate {
+            remove_field => [ "mine_lookup_result" ]
+          }
         }
       }
-
-
-      # Get the type of operation of interest
-
-      mutate {
-        add_field => { "[@metadata][action]" => "%{[nginx][access][method]} %{[nginx][access][url]}" }
+      if [nginx][access][referrer] {
+        translate {
+          field => "[nginx][access][referrer]"
+          destination => "[mds][group]"
+          dictionary_path => "/app/logstash/dictionaries/group.yml"
+          regex => true
+          fallback => 'other'
+          exact => true
+        }
       }
 
-      # Get the operation
-      translate {
-        field => "[@metadata][action]"
-        destination => "operation"
-        dictionary_path => "/app/logstash/dictionaries/action.yml"
-        regex => true
-        fallback => "other"
-      }
+    grok {
+      match => { "message" => ["%{DATA:[nginx][error][time]} \[%{DATA:[nginx][error][level]}\] %{NUMBER:[nginx][error][pid]}#%{NUMBER:[nginx][error][tid]}: (\*%{NUMBER:[nginx][error][connection_id]} )?%{GREEDYDATA:[nginx][error][message]}"] }
+      remove_field => "message"
+    }
+  }
+  else if [path] =~ "error" {
 
-      # Extract mine guid based on operation
-      if [operation] != "other" {
-        grok {
-          match => { "[nginx][access][url]" => [".*/api/mines/%{UUID:[mine][id]}", ".*/568/api/documents/mines/%{UUID:[mine][id]}"] }
-        }
-      }
+    mutate {
+      add_field => { "[log][type]" => "error" }
+    }
 
+    grok {
+      match => { "message" => ["%{DATA:[nginx][error][time]} \[%{DATA:[nginx][error][level]}\] %{NUMBER:[nginx][error][pid]}#%{NUMBER:[nginx][error][tid]}: (\*%{NUMBER:[nginx][error][connection_id]} )?%{GREEDYDATA:[nginx][error][message]}"] }
+      remove_field => "message"
     }
-    else if [fileset][name] == "error" {
-      grok {
-        match => { "message" => ["%{DATA:[nginx][error][time]} \[%{DATA:[nginx][error][level]}\] %{NUMBER:[nginx][error][pid]}#%{NUMBER:[nginx][error][tid]}: (\*%{NUMBER:[nginx][error][connection_id]} )?%{GREEDYDATA:[nginx][error][message]}"] }
-        remove_field => "message"
-      }
-      mutate {
-        rename => { "@timestamp" => "read_timestamp" }
-      }
-      date {
-        match => [ "[nginx][error][time]", "YYYY/MM/dd H:m:s" ]
-        remove_field => "[nginx][error][time]"
-      }
+    mutate {
+      rename => { "@timestamp" => "read_timestamp" }
+    }
+    date {
+      match => [ "[nginx][error][time]", "YYYY/MM/dd H:m:s" ]
+      remove_field => "[nginx][error][time]"
     }
   }
 }
 
-
 output {
-
-  stdout { codec => rubydebug}
+  
+  # stdout {}
```

Comment:
No we do not need it! Thanks

Reply:
I think we should leave the output on stdout just to look at the pod in case it starts behaving abruptly?

Do you think it will be a significant performance hit or something @jlaplante333 ?

Reply:
I have reenabled it. Thanks.